### ISSUE TITLE
Tree map

### ISSUE THREAD

**Issue by sarahmonster** [a8cteam51/just-one-tree#3]
We'll need to build a map that pulls from the registered lemon trees and displays them:

> The map was (for us) an expensive sub-project. It uses CARTO. We use colors to show
> density of planting in blocks because we found that people do not want their individual address
> shown on the map. I would like to be able to identify density by neighborhood because people
> are motivated by what their neighborhood is doing, but that seems challenging because there are not exact lines for neighborhood districts in San Francisco, although I've found a couple of pretty good maps. Also, some people might like to know how many lemon trees are on their block or their street. It would help us make the project more visible if we could do this, but I don't want to screw up the map that we have - it seems to be a technical specialty.

If we already have the data for the registered trees stored within the WordPress site as a custom post type (see #2) we'd just need to hook that data up to a map display. (I've used Mapbox in the past and it's quite nice.) We'd need to be careful to obscure addresses so they aren't too specific, and the neighbourhood functionality might require some thinking. 

The map isn't really highlighted on the current site; I could only find it by navigating to "register a tree": http://justonetree.org/got-a-tree-2/register-your-tree/

@Justonetree can you provide any additional information about your requirements for this map? From your comments it seems that the initial map is working for you, but you'd like to extend its functionality—what do you envision working best here?

**Comment by sarahmonster** [comment]
Quick note: There's neighbourhood information data in JSON format here: https://data.sfgov.org/Geographic-Locations-and-Boundaries/Planning-Neighborhood-Groups-Map/iacs-ws63/data but it doesn't match up with the neighbourhood boundaries that appear on the current map: http://justonetree.org/got-a-tree-2/register-your-tree/

**Comment by Justonetree** [comment]
Yes, that looks good. We used precinct boundaries which are not as good but remember we do not want to have individual addresses identified on the map

**Comment by sarahmonster** [comment]
Some conversations around this issue happened in Slack, so I'll do my best to summarise:

As a first step, @danielwrobert built a map using markerclusters (#78) but I'm a picky jerk and wanted the neighbourhood shaped shaded based on the number of trees in each of them (#79). 

I made a scale of colours related to the colours used on the site:
![colour-range](https://cloud.githubusercontent.com/assets/376315/24267929/d560da52-1003-11e7-8aef-c9ef651ddccc.png)

(.ai source in Dropbox if needed)

Given eight shades in total, we'll need to establish our legend/range of values. First I thought increments of 100 or 200 would be good, so
> < 100, 100-200, 200-300, 300-400, 400-500, 500-600, 600-700, 700+

But then I realised most neighbourhoods would be < 100 and thus the same colour. Using more exponential (reverse-exponential?) would give me colour variance:

> < 25, 25-50, 50-100, 100-250, 250-500, 500-750, 750-1000, 1000+
